packages/cli/src/utils/typescript.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path'
+import { existsSync } from 'fs'
+import chalk from 'chalk'
+import consola from 'consola'
+import { warningBox } from './formatting'
+
+export async function getTypescriptConfig(rootDir, extraOptions = {}) {
+  const tsConfigPath = path.resolve(rootDir, 'tsconfig.json')
+
+  if (!existsSync(tsConfigPath)) {
+    return undefined
+  }
+
+  consola.info('TypeScript environment detected')
```

Comment:
Isn't it better to explicitly say `tsconfig.json` found?

Reply:
just `consola.info('tsconfig.json found')` ?

Reply:
Yes. Or anything better. I just meant, `environment` term is little ambiguous to me.

Reply:
alright, will make it better :)